src/containers/Surah/index.js

```diff
@@ -433,30 +433,29 @@ class Surah extends Component {
           open={this.state.sidebarOpen}
           onSetOpen={(open) => this.setState({sidebarOpen: open})}
           styles={{sidebar: {zIndex: 9999}}}
-        >
-          <div className={`container-fluid ${style['surah-container']}`}>
```

Comment:
@mmahalwy Moving `surah-container` out of sidebar was necessary but introduces a new bug -
![screenshot at 2016-11-19 09-26-09](https://cloud.githubusercontent.com/assets/16819670/20452872/3d753aa6-ae3a-11e6-9f76-2b4e4a94e7c8.png)

This can be fixed by using `fixed` position for sidebar.